docs/subnets/customize-a-subnet.md

```diff
@@ -720,6 +838,6 @@ With `allowFeeRecipients` enabled, validators can specify their addresses to col
 
 :::warning
 
-If `allowFeeRecipients` feature is enabled on the Subnet, but a validator doesn't specify a "feeRecipient", the fees will be burned in blocks it produces.
+If `allowFeeRecipients` or FeeRewardManager is enabled on the Subnet, but a validator doesn't specify a "feeRecipient", the fees will be burned in blocks it produces.
```

Comment:
Is it called `rewardManagerConfig` (line 517) or `FeeRewardManager`?

Reply:
FeeRewardManager is the precompile name whereas `rewardManagerConfig` is the field name in the configuration files.